beacon-node-oapi.yaml

```diff
@@ -78,12 +78,8 @@ paths:
     $ref: "./apis/beacon/blocks/headers.yaml"
   /eth/v1/beacon/headers/{block_id}:
     $ref: "./apis/beacon/blocks/header.yaml"
-  /eth/v1/beacon/blocks:
-    $ref: "./apis/beacon/blocks/blocks.yaml"
```

Comment:
I don't think this should have been deleted. This API endpoint isn't deprecated and is the only way for validator clients to publish their non-blinded blocks to the beacon node

Reply:
hrm i may have deleted the wrong thing? will check now...

Reply:
you're correct, raising a pr to put back

Reply:
thanks!